### DISCUSSION TITLE
Bad Link In Docs: Use Pods Image Field As Thumbnail

### DISCUSSION THREAD

**Discussion by davidshq** [pods-framework/pods#6843] [Q&A]
Hi Folks,

FYI, the link on the above titled page:

http://blog.stefanxo.com/2013/08/display-thumbnails-with-pods/

Is no longer valid.

The Wayback Machine has a copy here:

http://web.archive.org/web/20160312035904/http://blog.stefanxo.com/2013/08/display-thumbnails-with-pods/

Though on my machine the code isn't visible unless highlighted.

Page with this link on it is:

http://pods.io/tutorials/use-pods-image-field-thumbnail/

Dave

**Comment by mastef** [comment]
@davidshq Link should work again
